### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by laisspportugal**
**Description**

This PR intends to create an initial setup for message-definition project. What is included:

- Renaming the project to point to lichtblick projects and references
- Change version on package.json
- Update needed dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,10 +21,10 @@
     "grammar"
   ],
   "author": {
-    "name": "Foxglove Technologies Inc",
-    "email": "contact@foxglove.dev"
+    "name": "Lichtblick",
+    "email": "lichtblick@bmwgroup.com"
   },
-  "homepage": "https://github.com/foxglove/message-definition",
+  "homepage": "https://github.com/lichtblick-suite",
```
Comment (aneuwald-ctw) [inline package.json:27]: It is missing the name of the repo. Was it on purpose?

**Review by aneuwald-ctw — DISMISSED** [review]
Just check my comment, if it was on purpose, it is approved

File: src/types.ts [hunk 2]
```diff
@@ -45,7 +45,7 @@ export type MessageDefinitionField = {
   /**
    * Type name of this field. Valid primitive type names will differ across each
    * interface definition language, and complex types are often user-defined.
-   * Examples: "string", "int32", "geometry_msgs/Point", "foxglove.Grid".
+   * Examples: "string", "int32", "geometry_msgs/Point", "lichtblick.Grid".
```
Comment (aneuwald-ctw) [inline src/types.ts]: This example should not be changed, as it is an schema and we don't have lichtblick schemas yet.

File: package.json (hunk 1 above)
Comment (laisspportugal) [reply package.json:27]: I used the same as lichtblick project, since the specific repo is already there on repository tag

**Review by aneuwald-ctw — APPROVED** [review]
LGTM

**Pull request merged**